packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -350,7 +359,11 @@ export class KeybindingWidget extends ReactWidget {
     protected renderRow(item: KeybindingItem, index: number): React.ReactNode {
         const { command, keybinding } = item;
         // TODO get rid of array functions in event handlers
-        return <tr className='kb-item-row' key={index} onDoubleClick={() => this.editKeybinding(item)}>
+        return <tr
+            className='kb-item-row'
+            key={index}
+            onDoubleClick={() => this.editKeybinding(item)}
```

Comment:
Please check react guideline about arrow functions in event handlers and performance

Reply:
The examples from [no-bind-fn-in-event-handlers](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#no-bind-fn-in-event-handlers) show versions where additional parameters are not used (which is not the case for the keybindings-widget). Should the code be refactored to not pass parameters, I am unable to get it to work properly by mixing the two.

Reply:
I guess in this case I should use `.bind`?
- https://reactjs.org/docs/handling-events.html#passing-arguments-to-event-handlers

I'll try it out!

Reply:
I tried using `.bind` but I see it is also not advised in our coding guidelines. I don't see how we can pass params and not create new function references. The guideline does not describe an example from which we also pass params to our event-handlers.